controllers/v1/solutionsController.js

```diff
@@ -1558,5 +1548,161 @@ module.exports = class Solutions extends Abstract {
 
         })
     }
+
+    /**
+    * @api {post} /assessment/api/v1/solutions/addEntities/:solutionId Add entity to solution
+    * @apiVersion 1.0.0
+    * @apiName Add entity to solution
+    * @apiGroup Solutions
+    * @apiParamExample {json} Request-Body:
+    * {
+    *	    "entities": ["5beaa888af0065f0e0a10515","5beaa888af0065f0e0a10516"]
+    * }
+    * @apiHeader {String} X-authenticated-user-token Authenticity token
```

Comment:
@aman-tunerlabs - Is internal access token required for this API?

Reply:
@aks30  yes we are calling this api in unnati